securedrop_client/api_jobs/downloads.py

```diff
@@ -148,13 +159,23 @@ def _decrypt_file(
         # server (e.g. spotless-tater-msg.gpg).
         filepath_in_datadir = os.path.join(self.data_dir, server_filename)
         shutil.move(file_path, filepath_in_datadir)
-        mark_file_as_downloaded(file_uuid, session)
+        mark_as_downloaded(type(db_object), db_object.uuid, session)
 
         try:
             self.gpg.decrypt_submission_or_reply(filepath_in_datadir, server_filename, is_doc=True)
+            # The file is stored on the filesystem so no need to set content in the database.
+            set_decryption_status_with_content(
+                model_type=type(db_object),
+                uuid=file_uuid,
+                is_decrypted=True,
+                session=session)
+            # Now that the file is decrypted, delete the encrypted file.
+            # os.remove(filepath_in_datadir)
```

Comment:
why commented?

Reply:
whoops this can be deleted since it was just an experiment with a different code change that I decided not to add